addons/sale_timesheet/tests/test_project.py

```diff
@@ -123,3 +124,7 @@ def test_compute_cost_in_employee_mappings(self):
         employee_mapping.cost = self.employee_user.timesheet_cost
         employee_mapping.employee_id = self.employee_company_B
         self.assertEqual(employee_mapping.cost, self.employee_company_B.timesheet_cost)
+
+    def test_open_product_form_with_default_service_policy(self):
+        form = Form(self.env['product.product'].with_context(default_service_policy='delivered_manual'))
+        self.assertEqual('delivered_manual', form.service_policy)
```

Comment:
I launch your test in master without your fix and it does not fail. 😅

Reply:
weird, it failing for me in 15.0
![image](https://github.com/odoo/odoo/assets/64937092/ea1a4973-49db-4f6c-b03c-b61a195c1459)